cypress/integration/banners/slack_notification_banner.ts

```diff
@@ -34,6 +30,11 @@ describe("Slack notification banner", () => {
   });
 
   it("does not show up after user has entered their username and clicks 'save'", () => {
+    cy.clearCookie("has-closed-slack-banner");
+
+    cy.visit("/version/5ecedafb562343215a7ff297/tasks");
+
+    cy.dataCy(slackNotificationBanner).should("be.visible");
```

Comment:
```suggestion
    cy.dataCy(slackNotificationBanner).should("exist");
```

Reply:
I think `be.visible` is better in this test since it's more specific than `exist`. Cypress makes the distinction between the two in v6. It's the first breaking change in this changelog: https://docs.cypress.io/guides/references/changelog#6-0-0